Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/CardHeader.cpp

```diff
@@ -214,6 +214,7 @@ namespace AzQtComponents
     void CardHeader::setExpandable(bool expandable)
     {
         m_expanderButton->setEnabled(expandable);
+        m_expanderButton->setVisible(expandable);
```

Comment:
Just want to confirm with UX that we want to show/hide the expander button for cards if it's expandable or not? @amzn-leenguy @rainbj

Reply:
Hey @cgalvan , Thanks or the tag. might need more context in the form like @hultonha suggested in recording of video or before after images. So i can give better feedback. 
Thanks!

Reply:
@ablepharus could you please attach a recording showing of the changes in action please? Thanks!

Reply:
I grepped the code, `setExpandable` was never called with `false` and the `m_expanderButton` is enabled by default.
So, there seems to be no change to existing widgets.

If you, as i do in the other commit, set a CardHeader as not expandable, the expander icon (that will not work) will be hidden.

Reply:
The Preset pane would look like this
![image](https://user-images.githubusercontent.com/57981/188502436-d1e9b531-5e1a-4b37-9c0b-ff801104cac2.png)